src/groups/bmq/bmqp/bmqp_ctrlmsg.xsd

```diff
@@ -1609,12 +1609,50 @@
       </documentation>
     </annotation>
     <choice>
+      <element name='authenticateRequest'      type='tns:AuthenticateRequest'/>
+      <element name='authenticateResponse'     type='tns:AuthenticateResponse'/>
```

Comment:
From our discussion today, we're leaning toward moving these types to a new, top-level complexType that just contains the authentication protocols.